src/app/editable-table/editable-table.component.html

```diff
@@ -0,0 +1,44 @@
+<mat-card>
+    <mat-card-content class="form-table">
+        <form [formGroup]="todoForm">
+            <ng-container formArrayName="todos">
```

Comment:
Is this one necessary? I would expect this to not be necessary since the `todos` are passed as data to the form table component, and each record within the form array is sent as context to ppw-column-cell.